### ISSUE TITLE
Bools throw error

### ISSUE THREAD

**Issue by AdamBrodzinski** [ScottONeal/xfdf#1]
Hello,

Recently i've ran into an issue where boolean fields throw this error:

```
Exception while invoking method 'fillManyPdfs' Error: addField() called, but no value argument supplied at XFDF.addField
```

```
  const xfdf = new Xfdf({translateBools: true})
    .fromJSON({fields: state.fillData })
    .generate();

```

and omitting them will make it work

```
// works

  const xfdf = new Xfdf({translateBools: true})
    .fromJSON({fields: _.omit(state.fillData, 'isCompleted', 'pmHasDownloaded', 'pmHasPrinted')  })
    .generate();

```

I've also tried it with no options and it's the same result. Any ideas?

**Comment by ScottONeal** [comment]
Thanks for the find and reporting it. I had made a fairly significant API change to xfdf at some point and must have broke Booleans! 

Should be fixed and added a test, see 34eb2d53e1

**Comment by AdamBrodzinski** [comment]
Awesome thanks for the quick fix!
